lib/esenalu.h

```diff
@@ -33,32 +33,30 @@ typedef enum ESENaluCodec {
   ESE_NALU_CODEC_H265,
 } ESENaluCodec;
 
-typedef enum
-{
-  ESE_H264_NAL_UNKNOWN      = 0,
-  ESE_H264_NAL_SLICE        = 1,
-  ESE_H264_NAL_SLICE_DPA    = 2,
-  ESE_H264_NAL_SLICE_DPB    = 3,
-  ESE_H264_NAL_SLICE_DPC    = 4,
-  ESE_H264_NAL_SLICE_IDR    = 5,
-  ESE_H264_NAL_SEI          = 6,
-  ESE_H264_NAL_SPS          = 7,
-  ESE_H264_NAL_PPS          = 8,
-  ESE_H264_NAL_AUD          = 9,
-  ESE_H264_NAL_SEQ_END      = 10,
-  ESE_H264_NAL_STREAM_END   = 11,
-  ESE_H264_NAL_FILLER_DATA  = 12,
-  ESE_H264_NAL_SPS_EXT      = 13,
-  ESE_H264_NAL_PREFIX_UNIT  = 14,
-  ESE_H264_NAL_SUBSET_SPS   = 15,
-  ESE_H264_NAL_DEPTH_SPS    = 16,
-  ESE_H264_NAL_SLICE_AUX    = 19,
-  ESE_H264_NAL_SLICE_EXT    = 20,
-  ESE_H264_NAL_SLICE_DEPTH  = 21
+typedef enum {
+  ESE_H264_NAL_UNKNOWN     = 0,
+  ESE_H264_NAL_SLICE       = 1,
+  ESE_H264_NAL_SLICE_DPA   = 2,
+  ESE_H264_NAL_SLICE_DPB   = 3,
+  ESE_H264_NAL_SLICE_DPC   = 4,
+  ESE_H264_NAL_SLICE_IDR   = 5,
+  ESE_H264_NAL_SEI         = 6,
+  ESE_H264_NAL_SPS         = 7,
+  ESE_H264_NAL_PPS         = 8,
+  ESE_H264_NAL_AUD         = 9,
+  ESE_H264_NAL_SEQ_END     = 10,
+  ESE_H264_NAL_STREAM_END  = 11,
+  ESE_H264_NAL_FILLER_DATA = 12,
+  ESE_H264_NAL_SPS_EXT     = 13,
+  ESE_H264_NAL_PREFIX_UNIT = 14,
+  ESE_H264_NAL_SUBSET_SPS  = 15,
+  ESE_H264_NAL_DEPTH_SPS   = 16,
+  ESE_H264_NAL_SLICE_AUX   = 19,
+  ESE_H264_NAL_SLICE_EXT   = 20,
+  ESE_H264_NAL_SLICE_DEPTH = 21
```

Comment:
this question is out of the scope of this mr, but why the explicit enumeration?

Reply:
because I wanted to start the parsin earlier :) I gonna remove it

Reply:
sorry it is used by the AUD alignement in h26xparse, so it is in use